src/techjoomla/tjnotifications/tjnotifications.php

```diff
@@ -164,14 +164,18 @@ public static function getRecipients($client,$key,$recipients,$options)
 	 *
 	 * @param   string  $body_template  A template body for email.
 	 * @param   array   $replacements   It is a object contains replacement.
+	 * @param   string  $client         A field same as component name.
 	 *
 	 * @return  string  $body
 	 *
 	 * @since 1.0
 	 */
-	public static function getBody($body_template, $replacements)
+	public static function getBody($body_template, $replacements, $client)
 	{
-		$matches = self::getTags($body_template);
+		$dispatcher = JEventDispatcher::getInstance();
+		$dispatcher->trigger('onBeforeMailSend', array(&$replacements,  &$body_template, $client));
```

Comment:
1. Remove extra spaces after `,`
2. Lets rename trigger as `onTjNotificationTemplatePrepare` to avoid potential tiger name conflicts
3. Add a wiki doc for how to use this

Reply:
@manojLondhe - onTjNotificationTemplatePrepare added